src/components/shared/refreshBar/refreshBar.js

```diff
@@ -1,22 +1,30 @@
 // Copyright (c) Microsoft. All rights reserved.
 
-import React from 'react';
+import React, { Component } from 'react';
 import moment from 'moment';
 import { Btn } from 'components/shared';
 import { svgs } from 'utilities';
 
 import './refreshBar.css';
 
-export const RefreshBar = (props) => (
-  <div className="last-updated-container">
-    {
-      props.isPending || props.time
-        ? <span className="time">
-            <span className="refresh-text">Last refreshed | </span>
-            { !props.isPending ? moment(props.time).format('MM/DD/YYYY, hh:mm:ss') : '--/--/----, --:--:--' }
-          </span>
-        : null
-    }
-    <Btn svg={svgs.refresh} className={`refresh-btn ${props.isPending ? 'refreshing' : ''}`} onClick={!props.isPending && props.refresh} />
-  </div>
-);
+export class RefreshBar extends Component {
+
+  refresh = () => !this.props.isPending && this.props.refresh();
+
+  render () {
+    const { t, isPending, time} = this.props;
+    return (
+      <div className="last-updated-container">
+        {
+          isPending || time
+            ? <span className="time">
+                <span className="refresh-text">{ isPending ? t('refreshBar.refreshing') : t('refreshBar.lastRefreshed')} | </span>
+                { !isPending ? moment(time).format('MM/DD/YYYY, hh:mm:ss') : '' }
```

Comment:
Use the constant like the last trigger renderer does.

Reply:
I'm leaving as is for now since it should be replaced by the component Molly made or the future version of it. That component will handle the function of a constant.